### PR TITLE
Build test utilities nuget package

### PR DESCRIPTION
**Pull request by drognanar**

### CODE REVIEW

**Comment by drognanar** [comment]
FYI @tannergooding @dotnet/roslyn-infrastructure

File: src/NuGet/Roslyn.VisualStudio.Test.Utilities.nuspec [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>Roslyn.VisualStudio.Test.Utilities</id>
+    <description>Roslyn VisualStudio Test Utilities.</description>
+    <dependencies>
+      <dependency id="Microsoft.VisualStudio.LanguageServices" version="[$version$]" />
```
Comment (jasonmalinowski) [inline src/NuGet/Roslyn.VisualStudio.Test.Utilities.nuspec]: This doesn't need to be a pinned version. Dropping the `[]` is sufficient here.

File: src/NuGet/Roslyn.VisualStudio.Test.Utilities.nuspec [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>Roslyn.VisualStudio.Test.Utilities</id>
+    <description>Roslyn VisualStudio Test Utilities.</description>
```
Comment (jasonmalinowski) [inline src/NuGet/Roslyn.VisualStudio.Test.Utilities.nuspec]: This seems...vague.

**Comment by drognanar** [comment]
Updated nuspec

**Comment by jasonmalinowski** [comment]
:+1:

**Pull request closed without merging**